src/en/clouds.md

```diff
@@ -39,6 +39,19 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region, see [Creating a controller](./controllers-creating.html).
```

Comment:
Ahh, I see he is out today. In that case, I'll merge as-is and we can revisit the location for the info again later.